### PR TITLE
Speed up dataset drawer rendering

### PR DESCRIPTION
**Pull request by maniarathi**

### CODE REVIEW

**Review by seve — COMMENTED** [review]
Great changes! The best type of coding is deleting code :) 

 Since there are now no async parts to the component we can refactor off of react-async for this component.

**Comment by maniarathi** [comment]
Heh, thanks :) Removed the async code!

**Review by seve — CHANGES_REQUESTED** [review]
InfoDrawer LGTM, but there are a few changes that can be propagated to InfoFormat.  All the skeleton references and the placeholders to fill the skeleton can be removed as the skeleton was only rendered when the component was waiting for async calls to complete.

**Comment by maniarathi** [comment]
Done! Thanks!

File: client/src/components/infoDrawer/infoFormat.js [hunk 1]
```diff
@@ -164,37 +157,22 @@ const renderLinks = (projectLinks, aboutURL, skeleton) => {
...
-    return [index, index];
-  }
-);
-
 const InfoFormat = React.memo(
   ({
     datasetTitle,
-    singleValueCategories = new Map(singleValueCategoriesPlaceholder),
+    singleValueCategories,
     aboutURL = "thisisabouthtelengthofaurl",
```
Comment (seve) [inline client/src/components/infoDrawer/infoFormat.js:174]:
```suggestion
    aboutURL,
```

File: client/src/components/infoDrawer/infoFormat.js [hunk 2]
```diff
@@ -164,37 +157,22 @@ const renderLinks = (projectLinks, aboutURL, skeleton) => {
...
     dataPortalProps = {},
-    skeleton = false,
   }) => {
     if (dataPortalProps.corpora_schema_version === "1.0.0") {
```
Comment (seve) [inline client/src/components/infoDrawer/infoFormat.js:177]:
```suggestion
    if (dataPortalProps.version.corpora_schema_version !== "1.0.0") {
```

**Review by seve — APPROVED** [review]
Small nit + bug we found!

File: client/src/components/infoDrawer/infoFormat.js (hunk 1 above)
Comment (maniarathi) [reply client/src/components/infoDrawer/infoFormat.js]: Done!

File: client/src/components/infoDrawer/infoFormat.js (hunk 2 above)
Comment (maniarathi) [reply client/src/components/infoDrawer/infoFormat.js]: Done!

**Pull request merged**